tests/util/kube_utils.go

```diff
@@ -111,6 +111,11 @@ func DeleteNamespace(n string, kubeconfig string) error {
 	return err
 }
 
+func DeleteDeployment(d string, n string, kubeconfig string) error {
```

Comment:
You need function comments I believe. Also nice to expand d and n to full name such as deployment and naemspace